mods/cnc/languages/chrome/en.ftl

```diff
@@ -233,96 +233,93 @@ supportpowers-support-powers-palette =
 
 button-command-bar-attack-move =
    .tooltip = Attack Move
-   .tooltipdesc = Selected units will move to the desired location
-    and attack any enemies they encounter en route.
+   .tooltipdesc = Selected units move to the target location
+    and engage any enemies they encounter en route.
 
-    Hold <(Ctrl)> while targeting to order an Assault Move
-    that attacks any units or structures encountered en route.
-
-    Left-click icon then right-click on target location.
+    Left-click the icon then right-click on the target.
+    Hold <(Ctrl)> while targeting to order an Assault Move,
+    that engages any units or structures encountered en route.
 
 button-command-bar-force-move =
    .tooltip = Force Move
-   .tooltipdesc = Selected units will move to the desired location
-     - Default activity for the target is suppressed
-     - Vehicles will attempt to crush enemies at the target location
-     - Helicopters will land at the target location
+   .tooltipdesc = Selected units move to the target location.
+     - Overrides default activity.
+     - Vehicles attempt to crush enemies at the target location.
+     - Helicopters land at the target location.
 
-    Left-click icon then right-click on target.
-    Hold <(Alt)> to activate temporarily while commanding units.
+    Left-click the icon then right-click on the target.
+    Hold <(Alt)> to temporarily activate while commanding units.
 
 button-command-bar-force-attack =
    .tooltip = Force Attack
-   .tooltipdesc = Selected units will attack the targeted unit or location
-     - Default activity for the target is suppressed
-     - Allows targeting of own or ally forces
-     - Long-range artillery units will always target the
-       location, ignoring units and buildings
+   .tooltipdesc = Selected units attack the targeted unit or location.
+     - Overrides default activity.
+     - Allows targeting of friendly or allied forces.
```

Comment:
The previous wording of "own or ally forces" implied a distinction between units under your control and those allied but not under your control. "friendly or allied forces" sounds clumsy in comparsion and lacks the same distinction betwen those units under your control and not.

Consider wording that restores the original distinction, alternatively shorten to just "allied forces" and stop making a distinction.